src/pages/wallets/[walletid].js

```diff
@@ -374,7 +374,9 @@ export default function Wallet(props) {
           sx={{ mt: [2.5, 5] }}
           variant="body2"
           dangerouslySetInnerHTML={{
-            __html: marked.parse(wallet.about || 'NO DATA YET'),
+            __html: marked.parse(
+              (wallet.about || 'NO DATA YET').replace(/\\n/g, '<br/>'),
```

Comment:
@frankienstan thank you for your work, because the `about` is in markdown format, so I don't think you should use html tag `<br/>` here, you should replace with newline `\n` and give it to `marked.parse` to generate the content (in html)

BTW, can paste a screenshot of the result?

Reply:
@dadiorchen  Yea sure ,
Before
![before](https://github.com/Greenstand/treetracker-web-map-client/assets/88585538/985e7c55-8a01-403b-8375-b1ee53ff7cb7)
After
![after](https://github.com/Greenstand/treetracker-web-map-client/assets/88585538/ffdfa3ca-5368-4011-9150-024fc354f4dd)

Reply:
Yea, the <br/> can do the work of breaking new line, but we don't need to do this by ourselves, we just need to fix the extra `\\n` for the string, and pass it to markdown lib, it will take care of all the work including the newline breaking

Reply:
we can replacing \\n with actual newline characters \n in the wallet.about content before passing it to marked.parse(). This way, marked.parse() will handle the Markdown rendering, including line breaks, as expected.

Reply:
@frankienstan yes, exactly